contracts/core/RebalancingSetToken.sol

```diff
@@ -212,13 +227,13 @@ contract RebalancingSetToken is
         require(block.timestamp >= proposalStartTime.add(proposalPeriod));
 
         // Create token arrays needed for auction
-        parseUnitArrays();
+        auctionSetUp();
 
         // Get core address
         address core = ISetFactory(factory).core();
 
         // Calculate remainingCurrentSets
-        remainingCurrentSets = unitShares.mul(totalSupply_);
+        remainingCurrentSets = unitShares.mul(totalSupply_).div(naturalUnit);
```

Comment:
It seems that making the calculation a function would makes things clearer and less error-prone. 

Maybe we have a function for this calculation already?

Reply:
I suppose we do on our other contracts but not on this one. Feels like over doing it to make it a function on this one where it wouldn't be used elsewhere. This wasn't an error, we just used to have the naturalUnit be 1 so there was no need to do the division, but with changing it to 10**10 it now needed to be included in the calculation.